src/__tests__/develop.spec.js

```diff
@@ -0,0 +1,22 @@
+import GetNumberSum from '../develop.js'
+function test() {
+  const testCases = [
+    { input: { a: 1, b: 2 }, expected: 3 },
+    { input: { a: -2, b: 1 }, expected: 0 },
+    { input: { a: 'not a number', b: 'number' }, expected: "Error: Cannot plus non number params" } 
+  ];
+  let testResults
+  testResults = testCases.map((testCase) => {
+  const results = GetNumberSum(testCase.input.a, testCase.input.b); 
+  const isPassed = results === testCase.expected; 
+  return isPassed;
+});
+  return testResults;
+}
+try{
+  if (results !== expected ) {
+    throw  Error("expexted !== result")};
```

Comment:
What exactly we want to return?

Reply:
I don’t quite understand what is considered an error in this test. Since the test itself shows us value errors of the type 2+2 !== 5 (false).
is the error a line that can interrupt code execution?

Reply:
Ah, i see the point
Ok. Just let me explaine a bit more. 
Error to throw is if we got any of test case `false`
Example: 
if we get [true, true, true] as the result of test() func we will print `OK`
If we get [true, false, true] as the result we need to find where exactly we got that false and throw an error

```suggestion
   if(result.hasSomeOf(false))
   throw "Threre is an error in test result."
```

Reply:
OK, thank you

Reply:
should I use results.includes?

Reply:
Up to you